src/NUnitFramework/nunitlite.tests/CommandLineTests.cs

```diff
@@ -522,9 +500,18 @@ public void NoResultSuppressesAllResultSpecifications()
             Assert.AreEqual(0, options.ResultOutputSpecifications.Count);
         }
 
-        #endregion
+        [Test]
+        public void InvalidResultSpecRecordsError()
+        {
+            var options = new NUnitLiteOptions("test.dll", "-result:userspecifed.xml;format=nunit2;format=nunit3");
+            Assert.That(options.ResultOutputSpecifications, Has.Exactly(1).Items
+                    .And.Exactly(1).Property(nameof(OutputSpecification.OutputPath)).EqualTo("TestResult.xml"));
+            Assert.That(options.ErrorMessages, Has.Exactly(1).Contains("invalid output spec").IgnoreCase);
+        }
+
+    #endregion
```

Comment:
The coding standards is currently empty regarding regions (see e.g. https://github.com/nunit/docs/wiki/Coding-Standards#use-of-regions) - and personally I don't like them :) - but I think that region pairs should be indented to the same level.